packages/nivo-line/src/LineSlices.js

```diff
@@ -45,6 +54,8 @@ LineSlices.propTypes = {
     showTooltip: PropTypes.func.isRequired,
     hideTooltip: PropTypes.func.isRequired,
     theme: PropTypes.object.isRequired,
+    tooltip: PropTypes.element,
```

Comment:
@jbenua Should it be either `element` or `func`? I am getting errors when I pass `tooltip` like this ```
tooltip={
              slice => {
                return (
                  <div>
                    <h2>{slice.id}</h2>
                    {slice.points.map((e, i) => (
                      <p key={i}>
                        <strong>{e.id}:</strong> {e.value}
                      </p>
                    ))}
                  </div>
                )
              }
            }
```